Docs/source/HardwareOverview/boards-sensors.rst

```diff
@@ -27,4 +27,42 @@ The reset button allows you to avoid powering the BPS off to reset it.
 .. note::
     You can configure the system to reset every time you use the Keil IDE to program it. 
 
-For more information, see the `STM32F413 datasheet <https://www.st.com/resource/en/reference_manual/dm00305666-stm32f413-423-advanced-arm-based-32-bit-mcus-stmicroelectronics.pdf>`__.
\ No newline at end of file
+For more information, see the `STM32F413 datasheet <https://www.st.com/resource/en/reference_manual/dm00305666-stm32f413-423-advanced-arm-based-32-bit-mcus-stmicroelectronics.pdf>`__.
+
+.. figure:: ../_static/leader-board-diagram.png
+    :align: center
+
+    STM32F413 bypass capacitors 
+
+The system's :abbr:`RTC (Real Time Clock)` is sourced from a low-power crystal oscillator. The crystal is used to set the :abbr:`PLL (Phase Lock Loop)`, which 
+sets the system core frequency (CPU clock speed). The controller supports a range of frequencies, but the default is 8MHz. 
+
+The :abbr:`MCU (Microcontroller Unit)` has an alternate internal clock that saves power consumption at the cost of precision. 
+Since this system is safety critical, the external 8MHz crystal was added. 
+
+.. note::
+    The internal and external clocks were not tested against each other.
+
+Capacitor values are set based on the crystal’s load capacitance and the capacitance of the whole board. 
+The crystal and capacitor should be as physically close to the MCU as possible to avoid signal drift.
+
+.. figure:: ../_static/crystal-capacitors.png
+    :align: center
+
+    STM32 crystal layout
+
+.. _power-connector:
+
+Power connector
+^^^^^^^^^^^^^^^
+The Leader board gets a 12V power supply. The voltage must be dropped to power the MCU and components. The total voltage required for all the components is +3.3V and +5V. 
```

Comment:
Why are there two total voltage values?

Reply:
We have a 12V power line that the BPS gets its power from. Some of the electronic components require 3.3V for power and some require 5V. The voltage needed to power each component is specified in their datasheet. We get 3.3V and 5V by converting 12V into 3.3V and 5V.